web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -121,6 +123,8 @@ export function Navigation() {
       const category = getCategoryForRoute(features, next);
       const previousCategory = getCategoryForRoute(features, location);
 
+      console.log('category', features, previousCategory);
```

Comment:
The same here, we're starting to port this back to master, so we should stop leaving debug code here.

Reply:
My bad, I forgot to do a sweep :D